pytest_mh/utils/fs.py

```diff
@@ -611,3 +611,27 @@ def chmod(self, mode: str, path: str, args: list[str] = []) -> SSHProcessResult:
         self.logger.info(f'Changing mode bits for "{path}" on {self.host.hostname}')
 
         return self.host.ssh.exec(["chmod", *args, mode, path])
+
+    def chown(
+        self, path: str, user: str | None = None, group: str | None = None, args: list[str] = []
```

Comment:
`[]` cannot be a default value, you must use `None` same for chmod - please fix it in another commit. (I need to find some python tool that will check this automatically.)

We already talked about this some time ago, do you remember? If not, I will explain.

Reply:
I forgot about it, but I remember now. Fixing